app/controllers/trains_controller.rb

```diff
@@ -7,14 +7,23 @@ def index
     @trains = TrainApi.new.index
   end
 
+  def create
+    response = TrainApi.new.create(params[:id], train_params)
```

Comment:
why do you need an id to create something?
how would you even have an id to create something in the first place?